docs/tutorial.rst

```diff
@@ -178,8 +205,6 @@ the training results for each training epoch.
    print("Finished training! You can move on to the next step!")
```

Comment:
Since it's a bash script, shouldn't we use printf or echo instead? print() throws me an error